lineup.js

```diff
@@ -1,9 +1,14 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((total, lineup) => {
```

Comment:
Totally minor but I would call this `player` it will cut down on confusion for folks reading the code. Technically `lineup` is the plural of `player`. Still works but would be an easier read.